lib/PuppeteerSharp/Media/PaperFormat.cs

```diff
@@ -20,56 +24,66 @@ private PaperFormat(decimal width, decimal height)
         /// Page width in inches
         /// </summary>
         /// <value>The width.</value>
-        public decimal Width { get; set; }
+        public readonly decimal Width;
```

Comment:
I wouldn't like to break compatibility for this. I would leave this as it is.
What we could do is turning PaperFormats into getters. By doing this we would return a new PaperFormat every time they request it.

Reply:
Great point about breaking compatibility.  I've reverted them back to getters and setters, and I've made the static properties getters instead of static properties.